whats_fresh/whats_fresh_api/views/entry/vendors.py

```diff
@@ -219,7 +220,7 @@ def vendor_list(request):
         'message': message,
         'new_url': reverse('new-vendor'),
         'new_text': "New Vendor",
-        'title': "All Vendors",
+        'title': get_fieldname.get_fieldname('vendors', 'vendors'),
```

Comment:
Does get_fieldname() require two parameters for any reason? What's being passed to it from the form?